index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <header class="toolbar">
+            <div class="my-logo hover-underline"><a href="index.html">My Logo</a></div>
+            <nav class="collapsed_menu">&#9776;</nav>
```

Comment:
- [Optional] Consider wrapping the hamburger icon in a button element because this is a button that opens the mobile menu when clicked.
```html
<nav>
  <button>&#9776;</button>
</nav>
```